src/pages/markets/components/markets/_commodities.js

```diff
@@ -9,10 +9,13 @@ import { commodities_options } from '../../static/content/_digital-options'
 import CFDs from '../sub-markets/_cfds'
 import DigitalOptions from '../sub-markets/_digital-options'
 import { Localize, localize } from 'components/localization'
+import { DerivStore } from 'store'
+
 const SimpleSteps = Loadable(() => import('components/custom/_simple-steps'))
 const OtherMarkets = Loadable(() => import('../sections/_other-markets.js'))
 
 const Commodities = ({ simple_step_content }) => {
+    const { is_eu_country, is_uk_country } = React.useContext(DerivStore)
```

Comment:
The old is `is_eu_country` and `is_uk_country` is used here to make a `is_row` condition, can you try using the new country  rule helper here?